docs/getting_started/folders_and_files/derivatives.md

```diff
@@ -184,6 +184,8 @@ BIDS Derivatives datasets are intended to be interpretable and distributable
 with or without the datasets used to generate them.
 This is necessary for storage and bandwidth constraints,
 as well as to permit the distribution of derivatives when the source data are restricted.
+Similarly, BIDS Raw datasets should be interpretable and distributable without
+all possible derivatives produced from them.
 
 This independence affords flexibility in the relative organization of datasets.
 The following examples show three ways to organize, relative to each other,
```

Comment:
> The following examples show three ways to organize, relative to each other,
> a raw BIDS dataset, a preprocessed derivative dataset, and an analysis that uses both as inputs.

GitHub won't let me apply the suggestion to the entire sentence, but I think we should delete the sentence and change it to the following:

```suggestion
The following examples show three ways to organize datasets:

1. A BIDS Raw dataset with derivative preprocessed and analysis data.
2. A BIDS Derivative dataset which uses both raw and preprocessed data as inputs.
3. A BIDS dataset with raw and derivative data.
```